charter.md

```diff
@@ -72,28 +78,151 @@ this works.
+community through the following channels:
+
+* the [community mailing list][dev-list]
+* the [Steering Committee public mailing list][steering-public-list]
+
+This notification will include:
+
+* a link to the aforementioned GitHub issue
+* the statement providing context on the reason for the vote
+
+There will be a period of two weeks for members of the community to reach
+out to Steering Committee members to provide feedback.
+
+Community members may provide feedback by the following methods:
+
+* commenting on the GitHub issue
+* replying to the notification email
+* sending an email to the
+  [Steering Committee private mailing list][steering-private-list]
+* sending a message to individual committee members
 
-Steering committee members are generally expected to attend every meeting. We
+After this feedback period, Steering Committee members must vote on the issue
+within 48 hours.
+
+If the vote of no confidence is passed, the member in question will be
+immediately removed from the committee.
+
+[dev-list]: mailto:dev@kubernetes.io
+[steering-private-list]: mailto:steering-private@kubernetes.io
+[steering-public-list]: mailto:steering@kubernetes.io
+[steering-repo]: https://git.k8s.io/steering
+
+## Voting
+
+In the course of the committee's operations, members will be expected to vote
+on decisions within the body's purview.
+
+These votes may be called on agreed-upon platforms by the committee, such as:
+
+* a pull request
+* an issue
+* a Steering Committee [meeting](#meetings)
+* a mailing list
+
+For public business, the vote must be captured on an issue or pull request.
+
+### Routine business
+
+Unless otherwise specified by a process, the requirement for passing a vote is
+a **_majority of the [fixed membership of the committee](#composition)_**.
+
+Example:
+
+* 7 (members) / 2 = 3.5
+* Round up to the nearest whole number (4)
+* 4 members would be required to pass a vote
+
+### Abstention
+
+For any self-elected reason, members of the committee may decide to abstain
+from a vote.
+
+Abstaining members will only be considered as contributing to quorum, in the
+event that a vote is called in a meeting.
+
+## Meetings
+
+Steering Committee members are generally expected to attend every meeting. We
 use the following guidelines to determine whether we have reached quorum and
 are able to proceed with a meeting.
 
-* Round quorum up if total number of members is odd
-* Quorum to meet is 1/2 members attending (eg: 6/12, 6/11, 4/7)
-* Quorum to vote in a meeting is 2/3 members attending (eg: 8/12, 8/11, 5/7)
+### Quorum
+
+Quorum **to meet** is a **_majority of the
+[fixed membership of the committee](#composition)_**.
+
+Example:
+
+* 7 (members) / 2 = 3.5
+* Round up to the nearest whole number (4)
+* 4 members in attendance would be required to meet
+
+Quorum **to vote in a meeting** is a **_two-thirds supermajority of the
+[fixed membership of the committee](#composition)_**.
+
+Example:
```

Comment:
Another example, suggest removal as per above.

Reply:
If/when these are removed, should update the PR description to remove ", with examples" in the Voting section bullet list.